ecocode-rules-specifications/src/main/assembly/swift.xml

```diff
@@ -0,0 +1,18 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.1.1"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.1.1 https://maven.apache.org/xsd/assembly-2.1.1.xsd">
+	<id>swift</id>
+	<formats>
+		<format>jar</format>
+	</formats>
+	<includeBaseDirectory>false</includeBaseDirectory>
+	<fileSets>
+		<fileSet>
+			<directory>${project.build.outputDirectory}</directory>
+			<includes>
+				<include>io/ecocode/rules/swift/*.*</include>
+			</includes>
+			<outputDirectory/>
+		</fileSet>
+	</fileSets>
+</assembly>
```

Comment:
il manque deux modifications du pom.xml afin de prendre en compte tous les rules SWIFT
cf https://github.com/green-code-initiative/ecoCode/blob/1ac339d27f14bc78b3af14192d1f67ac3a5943f5/ecocode-rules-specifications/pom.xml#L115

and cf https://github.com/green-code-initiative/ecoCode/blob/1ac339d27f14bc78b3af14192d1f67ac3a5943f5/ecocode-rules-specifications/pom.xml#L179

Reply:
@dedece35 je n'ai pas compris cette demande, peux-tu me dire en quoi elle consiste ?
Pour les autres langages, je ne vois pas ces deux attributs, donc je me demande pourquoi Swift devrait les avoir.